raspicam/README.md

```diff
@@ -1,29 +1,52 @@
-This folder contains some simple camera classification examples specific to Raspberry
-Pi, using the picamera python module to access the camera.
+# OpenCV examples for inferencing on Coral
 
-If you dont have picamera installed you can install it by:
+This folder contains example code using [picamera](https://github.com/waveform80/picamera) to obtain
+camera images and then perform image classification on the Edge TPU.
 
-```
-pip3 install picamera
```

Comment:
Seems more useful for people to use pip instead of pointing to the source code ? I worry some people will think they need to compile it.

Reply:
Huh? I replaced this with the command that runs this via the local shell script.